crates/snapbox/src/assert.rs

```diff
@@ -187,7 +187,7 @@ impl Assert {
 
         actual = actual
             .try_coerce(format)
-            .map_text(|s| self.normalize_text(s));
+            .map_text(|s| crate::utils::normalize_lines(s));
```

Comment:
I updated the [structured data issue](#92) to include `Normalize::Text` not just `Normalize::Newlines` so that we don't run into this issue again.